torchbenchmark/util/experiment/metrics.py

```diff
@@ -42,13 +44,10 @@ def maybe_synchronize(device: str):
 
 def get_latencies(
     func, device: str, nwarmup=WARMUP_ROUNDS, num_iter=BENCHMARK_ITERS
-) -> List[float]:
+) -> Tuple[List[float], List[float]]:
```

Comment:
I'm concerned that this PR introduces too significant a change to the core APIs. As an alternative, can you consider adding an option to skip the warmup phase and use the actual run results as the 'warmup' results?